packages/amplication-data-service-generator/src/server/resource/controller/create-controller.ts

```diff
@@ -247,7 +276,47 @@ async function createToManyRelationMethods(
     UPDATE: builders.identifier(camelCase(`update ${field.name}`)),
     UPDATE_PATH: builders.stringLiteral(`/:id/${field.name}`),
     SELECT: createSelect(relatedEntityDTOs.entity, relatedEntity),
+  };
+
+  interpolate(toManyFile, toManyMapping);
+
+  const classDeclaration = getClassDeclarationById(
+    toManyFile,
+    TO_MANY_MIXIN_ID
+  );
+
+  /** when an object has two identical keys, the first one gets override by the latter
+   * on toManyMapping we have 2 methods ids that their action ie 'update' (update and connect)
+   * In object the key must be a string or symbol. Therefore,
+   * I used Map because there is no way to use the method id as the key and the action as a value
+   **/
+  const toManyMethodsIdsActionPairs = new Map();
+
+  toManyMethodsIdsActionPairs.set(
+    toManyMapping["FIND_MANY"] as namedTypes.Identifier,
+    EnumEntityAction.Search
+  );
+  toManyMethodsIdsActionPairs.set(
+    toManyMapping["UPDATE"] as namedTypes.Identifier,
+    EnumEntityAction.Update
+  );
+  toManyMethodsIdsActionPairs.set(
+    toManyMapping["CONNECT"] as namedTypes.Identifier,
+    EnumEntityAction.Update
+  );
+  toManyMethodsIdsActionPairs.set(
+    toManyMapping["DISCONNECT"] as namedTypes.Identifier,
+    EnumEntityAction.Delete
+  );
```

Comment:
why are you using Map here and object in the parent use cases? 
if you decide to go with Map, please use Map< EnumEntityAction, namedTypes.Identifier>
if you decide to use an object, you should also consider using a type for the object instead of using `as`